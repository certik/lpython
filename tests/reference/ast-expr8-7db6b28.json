{
    "basename": "ast-expr8-7db6b28",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/expr8.py",
    "infile_hash": "ae545a93dbe13ae732ffb49c1c8acd8bcd03ba15e7d6b9cc2b420dec",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-expr8-7db6b28.stdout",
    "stdout_hash": "130ad60e40a60b4ab50843f768279b14ef54ff39be059b712a4c6eda",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}