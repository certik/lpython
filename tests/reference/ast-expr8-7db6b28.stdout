(Module [(FunctionDef test_binop ([] [] [] [] [] [] []) [(AnnAssign (Name x Store) (Name i32 Load) () 1) (AnnAssign (Name x2 Store) (Name f32 Load) () 1) (Assign [(Name x Store)] (BinOp (ConstantInt 2 ()) Pow (ConstantInt 3 ())) ()) (Assign [(Name x2 Store)] (BinOp (ConstantInt 2 ()) Pow (ConstantFloat 3.500000 ())) ()) (Assign [(Name x Store)] (BinOp (ConstantInt 54 ()) Sub (ConstantInt 100 ())) ()) (Assign [(Name x2 Store)] (BinOp (BinOp (ConstantFloat 3.454000 ()) Sub (ConstantFloat 765.430000 ())) Add (ConstantFloat 534.600000 ())) ()) (Assign [(Name x2 Store)] (BinOp (ConstantFloat 5346.565000 ()) Mult (ConstantFloat 3.450000 ())) ()) (Assign [(Name x2 Store)] (BinOp (ConstantFloat 5346.565000 ()) Pow (ConstantFloat 3.450000 ())) ())] [] () ())] [])
